src/sybil/column_store_test.go

```diff
@@ -98,7 +106,7 @@ func TestColumnStoreFileNames(t *testing.T) {
 		t.Error("Found other records than rowblock")
 	}
 
-	nt.DigestRecords(minFilesToDigest)
+	nt.DigestRecords(minFilesToDigest, digestSpec)
```

Comment:
should minFilesToDigest be pushed into digestSpec?

Reply:
I did this at first but it felt a bit awkward as it's returned from IngestRecords which accepts a digestSpec.